core/templates/components/state-editor/state-editor-properties-services/state-property.service.spec.ts

```diff
@@ -17,313 +17,98 @@
  *   editor page.
  */
 
-import { fakeAsync, TestBed, tick } from '@angular/core/testing';
```

Comment:
This file was containing redundant tests blocking stricter type checking for other files dependent on the service.